devops/nightly.yml

```diff
@@ -39,4 +39,3 @@ jobs:
 
 - template: templates/build-widget-job-template.yml
 
-- template: templates/limited-installation-job-template.yml
```

Comment:
Why is this being removed?

Reply:
Because "limited" = no matplotlib, so since he added it as a "core" dependency it's not needed anymore.